src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -27,7 +27,7 @@ namespace OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation
     /// </summary>
     internal static class AWSLambdaUtils
     {
-        internal const string ActivitySourceName = "Amazon.AWS.AWSLambdaInstrumentation";
+        internal static string ActivitySourceName = typeof(AWSLambdaUtils).Assembly.GetName().Name;
```

Comment:
Can we have here `OpenTelemetry.Instrumentation.AWSLambda`?
Current assembly name is temporary, until 2.0 release.

Reply:
You mean hardcoded, until the release of 2.0?

Reply:
Yes, to avoid further change in ActivitySourceName